src/main/java/com/corndel/pixmate/drawings/Checkboard.java

```diff
@@ -0,0 +1,33 @@
+package com.corndel.pixmate.drawings;
+
+public class Checkboard {
+    public static void main(String[] args) {
+        /*
+         * * * * *
```

Comment:
I like that you have visualized each of the shapes you will need to draw. If you are iterating through data, writing down what you want to happen at each stage can definitely help you out.